data/json/npcs/Backgrounds/OtherSurvivorStories/Lapin_01.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "BOSS_Lapin_01_Story1",
+    "type": "talk_topic",
+    "dynamic_line": "Actually, yeah.  It's a long shot but we could follow it if you like.  I ran into another survivor once - don't think she made it, she was heading to a nearby mall, and that place was overrun.  She mentioned seeing a guy taking care of his cabin in the woods though.  Said she thought about checking on him, but that was when things were all going nuts, and she decided against it.  I got some directions and never followed through on it.",
+    "responses": [
+      {
+        "text": "Let's see how to get there and go check on the guy.",
+        "topic": "BOSS_Lapin_01_Story2",
+        "condition": { "npc_has_trait": "BOSS_No_Others_02" },
+      },
+        "text": "<done_conversation>",
+        "topic": "TALK_FRIEND"
+      },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  }
+]
```

Comment:
`[
  {
    "id": "BOSS_Lapin_01_Story1",
    "type": "talk_topic",
    "dynamic_line": "Actually, yeah.  It's a long shot but we could follow it if you like.  I ran into another survivor once - don't think she made it, she was heading to a nearby mall, and that place was overrun.  She mentioned seeing a guy taking care of his cabin in the woods though.  Said she thought about checking on him, but that was when things were all going nuts, and she decided against it.  I got some directions and never followed through on it.",
    "responses": [
      {
        "text": "Let's see how to get there and go check on the guy.",
        "topic": "BOSS_Lapin_01_Story2",
        "condition": { "npc_has_trait": "BOSS_No_Others_02" }
      },
      { "text": "<done_conversation>", "topic": "TALK_FRIEND" },
      { "text": "<end_talking>", "topic": "TALK_DONE" }
    ]
  }
]
`